packages/astro/src/types/public/config.ts

```diff
@@ -2458,6 +2458,31 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 		 * See the [experimental raw environment variables guide](https://docs.astro.build/en/reference/experimental-flags/raw-env-values/) for more information.
 		 */
 		rawEnvValues?: boolean;
+		/**
+		 * @name experimental.chromeDevtoolsWorkspace
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 5.13
+		 * @description
+		 *
+		 * Enables Chrome DevTools workspace integration for the Astro dev server.
+		 *
+		 * When enabled, the dev server will automatically configure a Chrome DevTools workspace for your project,
```

Comment:
```suggestion
		 * When enabled, the dev server will automatically configure a [Chrome DevTools workspace](https://developer.chrome.com/docs/devtools/workspaces) for your project,
```
Let's inline this link to define what the workspaces are, and let the "see more info" be the link to our own experimental docs, where we are free to add more specific Astro stuff as we need.